### DISCUSSION TITLE
Remap Mouse Turbo Clicker to other keys

### DISCUSSION THREAD

**Discussion by frgmnt** [getreuer/qmk-keymap#23] [Q&A] [answered]
Is there a simple way to reuse the script for the Mouse Turbo Clicker to function with specific keys when held down? I'm looking to have a similar turbo function on my space and G keys, but I'm not too versed in code to fully understand how the script works. Do I create custom keycodes for space and G inside "keymap.c", then set those custom keycodes to the layout positions where those keys would normally be on my keyboard? Sorry my coding experience is very minimal.

**Comment by getreuer** [comment, marked as answer]
That's an interesting use case. Thanks for this idea. To repeatedly click another key, you can change the keycode sent in Mouse Turbo Click's callback . I just [pushed a commit](https://github.com/getreuer/qmk-keymap/commit/9cb047dc3ff10298ae08f657ac73e949fb46f323) to make this a bit easier: define `MOUSE_TURBO_CLICK_KEY` in your config.h as the key to be repeatedly clicked.

In config.h:

```c
// Repeatedly click the Space key.
#define MOUSE_TURBO_CLICK_KEY KC_SPC
```

As it is, Mouse Turbo Click is limited to manipulating a single key, so unfortunately there isn't a direct way to use it to do turbo pressing for both Space and G. I'll think about whether there's a way to expand this library to turbo pressing keys more generally while hopefully keeping it simple enough to use.
